_posts/2024-10-16-djangocon-us-call-for-venue-proposal-2025-extension.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title:  "DjangoCon US: Call for Venue Proposal 2025 extension"
+date: 2024-10-16 12:00:00
+categories: ""
+author_name : DEFNA
+author_url : /about
+author_avatar: defna
+show_avatar : true
+read_time : ""
+feature_image: 2016-06-08
+show_related_posts: false
+permalink: /announcements/2024/10/16/djangocon-us-call-for-venue-proposal-2025-extension/
+---
+
+First off, thank you to the teams who sent in proposals. We're very appreciate of the time and effort that you've put in to this point.
+
+However, since we have yet to find a proposal that meets all of our needs we're officially re-opening our call of venue proposals.
+
+If you are interested in bringing DjangoCon US to your city for 2025 and/or 2026, please submit your proposal to [hello@defna.org](mailto:hello@defna.org) no later than November 15, 2024.
+
+For 2025, we are looking at conference dates of October 5-10, 2025 or October 12-17, 2025.
```

Comment:
The DSF got upset with us for conflicting with Sukkot in 2022 after they signed off on it. I'd like to avoid the latter week if possible to dodge that a second time.